logging/proto_conversions_test.go

```diff
@@ -141,6 +179,18 @@ func TestFieldConversion(t *testing.T) {
 			test.That(t, err, test.ShouldBeNil)
 			test.That(t, key, test.ShouldEqual, tc.field.Key)
 			test.That(t, val, test.ShouldResemble, tc.expectedVal)
+
+			zapField, err := FieldFromProto(field)
+			test.That(t, err, test.ShouldBeNil)
+			if tc.expectedField.Type != zapcore.UnknownType {
```

Comment:
I'm guessing `UnknownType` is the default value for a default `expectedField` (i.e: most test cases)?

Reply:
Correct. It's the default value and should never actually be set at runtime because zap will panic in the json serializer (and possibly other places) if it encounters a field with `UnknownType`